client/app/styles/base.styl

```diff
@@ -0,0 +1,91 @@
+@font-face
```

Comment:
`font-face` declaration isn't required as the page already load the global `font.css` file that contains all fonts declarations. (see #559)